### DISCUSSION TITLE
could not write config file

### DISCUSSION THREAD

**Discussion by WebDevRobert** [OSNVR/OS-NVR#1] [Q&A]
user@machine:~$ docker --version
Docker version 23.0.4, build f480fb1

user@machine:~$ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.2 LTS"

user@machine:~$ sudo docker run -it \
	--env TZ=America/New_York \
	--shm-size=500m \
	-v /home/_nvr/os-nvr/configs \
	-v /home/_nvr/os-nvr/storage \
	-p 2000:2000 \
	osnvr/os-nvr_bundle:latest

The above comes to a screeching halt with the following error:

Generating configs/env.yaml and demo monitor.
2023/04/20 06:04:33 start: could not generate config file: could not write config file: open /home/_nvr/os-nvr/configs/env.yaml: permission denied
exit status 1

**Comment by WebDevRobert** [comment]
Additionally, I've tried these same steps using sudo and while logged in as root. I get the same error either way.

**Comment by Curid** [comment]
https://codeberg.org/Curid/os-nvr/issues/293
